core/dbt/adapters/TODO.md

```diff
@@ -0,0 +1 @@
+Move the warehouse adapters to a subfolder of adapters (adapters/warehouse).
```

Comment:
These are in `plugins/` right now (and three of them are soon to leave this repo). Let's think about whether `local_filesystem` makes more sense under `plugins/` or as part of `core/`.

Also... we'll need to come up with a new word for this, since none of warehouse, database, query engine, etc is quite perfect. To date, I've just been using "adapter," but obviously we've got adapters for more things now :)

Reply:
@jerco That makes sense.  I'll do some folder juggling and see where we land.  As far as naming goes.. I've been leaning towards "warehouse adapters" and "storage adapters", but I don't hold that opinion too strongly.

Reply:
lmk if you want to chat plugins! I think I have a good sense how that could look